### PR TITLE
feat(config): Configurable presentation pointer size

### PR DESCRIPTION
**Pull request by hiroshisuga**
### What does this PR do?

Enables users to change the size of presentation pointer and sets the default value 10 in diameter, which is what BBB 2.5 adopted.

### Motivation

Starting lectures using BBB 2.6, I realised that the presentation (cursor) pointer is way smaller than it has been until 2.5. Not sure if it's intended change or confusion of radius with diameter, but I prefer a larger pointer. For those who prefer different size, I set this value configurable. 

### More

Until 2.5:
スクリーンショット 2023-06-07 22 41 01

On 2.6:
スクリーンショット 2023-06-07 22 44 00

After the change (configurable at setting.yml):
スクリーンショット 2023-06-07 22 41 34

### CODE REVIEW

File: bigbluebutton-html5/private/config/settings.yml [hunk 1]
```diff
@@ -759,6 +759,7 @@ public:
   whiteboard:
     annotationsQueueProcessInterval: 60
     cursorInterval: 150
+    pointerDiameter: 10
```
Comment (antobinary) [inline bigbluebutton-html5/private/config/settings.yml:762]: If this was kept as `5`, I think the pull request would just be making the size configurable.

Given that you're changing the size, we get to talk about should we mirror the change on playback or video format...

**Comment by antobinary** [comment]
Thank you for this pull request, @hiroshisuga !

File: bigbluebutton-html5/private/config/settings.yml (hunk 1 above)
Comment (hiroshisuga) [reply bigbluebutton-html5/private/config/settings.yml]: Done! Thanks for the suggestion.

**Comment by hiroshisuga** [comment]
The change done.

Yes, I remember the discussion in #13898. I wonder if the pointer in a real presentation and those in the playback or video are need to be the same... playback is playback, which does not anyway reproduce what happens in the real presentation. 

The trouble to me is that when I show a BBB presentation by a projector in a face-to-face realtime lecture (with some online audience), the pointer cursor is too small to be clearly visible from the back of classroom. For the playback, students would watch it in a PC screen and would never miss the pointer even if it's that small.

As always, I am fine with whatever the devs decide.

**Review by antobinary — APPROVED** [review]
LGTM!

**Review by ramonlsouza — APPROVED** [review]

**Pull request merged**